addons/pos_loyalty/static/src/overrides/models/loyalty.js

```diff
@@ -1127,15 +1127,19 @@ patch(Order.prototype, {
      */
     _getCheapestLine() {
```

Comment:
if you feel like it this method could use some help :) something like `Math.min(...mylines.filter())` would be nice

Reply:
I tried to make it a bit cleaner, I think it's a bit better like this